app/views/request/show.html.erb

```diff
@@ -165,6 +165,17 @@
     </div>
   <% end %>
 
+  <% unless @render_to_file %>
```

Comment:
Unfortunately, the dropdown to the bottom left of the page doesn't quite work for me (screenshots are themeless to make sure my theme changes weren't causing this).

### Safari (Mac)
![screen shot 2017-01-11 at 18 08 06](https://cloud.githubusercontent.com/assets/27760/21860556/15bcbbd8-d829-11e6-8d1a-9185b2a92152.png)

(the dark grey shadow to the left is the window behind the browser)

### Chrome (Mac)

![screen shot 2017-01-11 at 18 16 20](https://cloud.githubusercontent.com/assets/27760/21860812/39c2bb80-d82a-11e6-972e-be731d7b05ac.png)

(screen grab includes space to the left and underneath the browser window)